### ISSUE TITLE
Version CSS pour les malvoyants

### ISSUE THREAD

**Issue by david-saisondor** [LAB-MI/attestation-deplacement-derogatoire-q4-2020#99]
Bonjour,
Il manque la possibilité d'activer une version contrastée de la page (fond noir et typographie blanche et bold) à l'usage des personnes malvoyantes.

**Comment by coralieco** [comment]
Bonjour,
Je veux bien regarder ce point ! 
Je viens de tester la page avec [Color Contrast Accessibility Validator](https://color.a11y.com/Contrast/) qui certifie qu'il n'y a aucun problème de contraste. Avez-vous d'autres outils à me conseiller pour me guider dans l'amélioration de l'accessibilité ?
Merci !

**Comment by david-saisondor** [comment]
Effectivement, au niveau contraste, la page est conforme. Toutefois, il existe des personnes photosensibles qui ne sont pas en mesure de soutenir du regard le fond blanc d'un écran. Comme remède à ce problème, il existe des thèmes "dark" dont l'activation est implémentée dans un nombre croissant d'outils.
En ce qui concerne la page du formulaire, cette librairie pourrait convenir. En outre, elle est simple à mettre en oeuvre : [https://github.com/sandoche/Darkmode.js](https://github.com/sandoche/Darkmode.js)

**Comment by coralieco** [comment]
Ok merci ! Ca serait pour la version web de l'application ? 

Car il me semble que le dark mode peut être activé au niveau des navigateurs (exemple sur mon Android). Ou via des add-ons.
![signal-2020-11-03-dark-mode](https://user-images.githubusercontent.com/22852543/97993823-dea47180-1de4-11eb-87d0-dd98f701f3f0.png)

**Comment by david-saisondor** [comment]
Oui, spécifiquement en version desktop.

**Comment by coralieco** [comment]
Je travaille sur une version dark mode activable sur la page, mais j'ai peur que ça fasse doublon avec ce qui existe déjà car chaque navigateur propose un mode sombre (même sur desktop) via des extensions (Dark Mode pour Chrome et Firefox par exemple). Je ne connais rien à ce sujet, mais j'image que les personnes malvoyantes utilisent déjà ces extensions par défaut. 

Donc je ne sais pas trop si je dois continuer sur ma PR ou non. Qu'en pensez-vous ? 
Sinon, probablement que le dark mode des navigateurs par défaut peut être amélioré en terme de contraste (voir printscreen ci-dessous). 

![image](https://user-images.githubusercontent.com/22852543/98023384-25a35e80-1e07-11eb-898a-6494a617eda5.png)

**Comment by a2br** [comment]
C'est dommage que les navigateurs ne proposent pas de paramètre pour passer du light mode au dark mode comme sur mobile, ça te simplifierait la tâche...
Pour que ça ne rentre pas en conflit avec les modes natifs et si ta cible est l'accessibilité, alors peut être que tu devrais continuer sur ta PR, mais en tant que 'Mode accessibilité visuelle' qui override les modes classiques, peut-être activable dans le footer ou en haut de la page.
Ce n'est qu'une suggestion, fais-en ce que tu veux après

**Comment by david-saisondor** [comment]
Pour ma part, je pense que si ce n'est pas natif, que cela nécessite des addons, alors ce n'est pas accessible. Cela revient à s'en remettre aux dévelopeurs d'addons pour rendre son application acessible.
Je reste donc campé sur ma position première : s'il y a un commutateur dark/light, cela ne rentera pas en conflit avec les thèmes des navigateurs. Car en toute logique, si le navigateur est en thème 'dark', alors le commutateur passera la page en thème 'light'. Ça n'est pas vraiment de l'override, mais de la simple commutation ; Le plugin proposé (https://github.com/sandoche/Darkmode.js) se contentant de faire du négatif des teintes paramétrées dans le CSS.
Vu la simplicité de mise en oeuvre, je pense que cela mériterait d'être essayé.

**Comment by a2br** [comment]
@david-saisondor Peut-être sur PC, mais le dark mode est très accessible sur téléphone. Je pense qu'un mode haut contraste serait plus intéressant qu'un switch; après c'est une option intéressante, ça vaudrait sans doute l'ouverture d'une autre issue.

**Comment by coralieco** [comment]
C'est bien la librairie https://github.com/sandoche/Darkmode.js que j'ai utilisée dans ma PR. Effectivement, les couleurs dark sont plus contrastées.
